packages/runtime/client/rest/test/endpoints/contracts.spec.e2e.ts

```diff
@@ -0,0 +1,26 @@
+
+import { pipe } from 'fp-ts/lib/function.js'
+import * as TE from 'fp-ts/lib/TaskEither.js';
+import * as O from 'fp-ts/lib/Option.js';
+
+import { mkRestClient } from '@marlowe.io/runtime-rest-client/index.js';
+
+import { getMarloweRuntimeUrl } from '../context.js';
+
+import console from "console"
+global.console = console
```

Comment:
This seems weird, a little explanation on what and why you did this can go a long way.